### DISCUSSION TITLE
Filament::notify() displaying twice

### DISCUSSION THREAD

**Discussion by DSpeichert** [filamentphp/filament#2316] [Help]
I can't figure it out, but when using `Filament::notify('success', 'test')` in a LiveWire method, the message is displayed twice - once when it should and then on next page load as well.

Any ideas of what could be wrong? Same notifications as part of Admin Panel's stock CreateRecord work fine.

**Comment by danharrin** [comment]
Can you `dump()` something in the Livewire method to check if its run twice?

**Reply by DSpeichert** [reply]
I did, only shows once. The dump comes back with livewire notification:
> ^ "test"
{"effects":{"html":null,"dispatches":[{"event":"notify","data":[{"id":"62771b212e4ae","status":"success","message":"Updated."}]}],"dirty":[]},"serverMemo":{"checksum":"2a8a0a601b6deb4aed5d78f5146e2db4adebc366c88a67de1b3d6acc472db71d"}}

Telescope shows that session contains the notification for that request, which is not the case with Admin Panel's CRUD operation (the notification that appears only once).

It seems that for whatever reason, the notification is both sent and pushed to the session (and sent next time). Any ideas why that could be the case?

**Reply by danharrin** [reply]
Fixed this in #2341
